commands/initial/templates/e2e/example.e2e-spec.ts

```diff
@@ -0,0 +1,10 @@
+import {examplePo} from './page-objects/example.po';
+
+describe('Example Page', () => {
+
+    beforeEach(() => examplePo.go());
+
+    it('should welcome user to the example page', () => {
+        expect(examplePo.welcome.getText()).toBe('Welcome');
```

Comment:
I know what you mean but they come with an advantage and at a cost: 
The cost is that what you want to get rid of, the example app/spec/whatever. 
The advantage is that your setup is already ready to go, you just need to overwrite the one test case you wanted to get rid off.
But I actually understand that for some who is like generating the 30th component he doesn't want to delete the example stuff all over again. That's why I would suggest adding a flag to every of your generating-component/directive/service/pipe script that is boolean and is named "--with--example" (or --without--example).

If you use something like [rc](https://www.npmjs.com/package/rc) then you can even use a configuration file that behaves like I described above (CLI > configuration > ...)